### PR TITLE
Documentation Change - Fixes #92

### LINKED ISSUE #92: Write guide about storage options
github - commit link
drive, etc...

### PR DESCRIPTION
**Pull request by Debaditya-Som**
---
 Documentation Change - Fixes #92
---

**Description:**

This PR addresses issue #92 by making modifications to the README.md file. The changes primarily focus on enhancing the documentation for better clarity and understanding.

**Changes Made:**

- Modified the README.md file.
- Enhanced the appearance of the README.md file  
- Included information on preparing data, running the Databus server, publishing the first artifact, and querying metadata

**Context:**

Issue #92 highlighted the need for improved documentation regarding storage options in the README.md file. This PR aims to fulfill that need by providing a comprehensive guide on storage options within the README.md file.

**Related Issues:**

- Closes #92 

**Checklist:**

- [ ] The documentation changes have been reviewed and approved.
- [x] The PR has been tested for accuracy.
- [x] All checks have passed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,42 +1,53 @@
 ---
-description: >-
+Description: >-
  DBpedia Databus is a transformative platform for agile data integration, collaboration, and automation via a structured metadata Knowledge Graph.
 ---
 
-Important Links: [Gitbook Docu](https://dbpedia.gitbook.io/databus/), [Join Discord](https://discord.gg/fB8byAPP7e),  [Databus used in the DBpedia Project](https://databus.dbpedia.org/)
+## Important Links
```
Comment (Debaditya-Som) [inline README.md:6]: Arranged the provided links in form of bullet points

**Comment by manonthegithub** [comment]
Thank you for sending pull request! We will review asap and merge if agree with the changes :). Our team atm is quite busy, so it may take more than a week!

File: README.md [hunk 2]
```diff
@@ -113,16 +125,17 @@ icon and then `Publish Data`.
 
 Fill in the form for publishing and submit. 
 For simplicity, you can enter any name for group, artifact and version.
-Use the URI of the file we prepared for publishing (`https://raw.githubusercontent.com/dbpedia/databus/68f976e29e2db15472f1b664a6fd5807b88d1370/README.md`) 
+Use the URI of the file we prepared for publishing [here](https://raw.githubusercontent.com/dbpedia/databus/68f976e29e2db15472f1b664a6fd5807b88d1370/README.md)
```
Comment (manonthegithub) [inline README.md:128]: This I would keep as it was before, it is supposed to be used for copy/paste and not for clicking on it

File: README.md [hunk 3]
```diff
@@ -113,16 +125,17 @@ icon and then `Publish Data`.
...
-Use the URI of the file we prepared for publishing (`https://raw.githubusercontent.com/dbpedia/databus/68f976e29e2db15472f1b664a6fd5807b88d1370/README.md`) 
+Use the URI of the file we prepared for publishing [here](https://raw.githubusercontent.com/dbpedia/databus/68f976e29e2db15472f1b664a6fd5807b88d1370/README.md) 
 in the `Files` section.  
 
 After publishing the data should be visible on  `account icon -> My Account -> Data tab`.
 
 ℹ️ Notes:
-* See more on how to organise your data [here](https://dbpedia.gitbook.io/databus/model), there you can find detailed explanations and our suggestions on structuring your datasets.
-* For automated metadata uploads see the [API](https://dbpedia.gitbook.io/databus/usage/api)
+- See more on how to organise your data [here](https://dbpedia.gitbook.io/databus/model), there you can find detailed explanations and our suggestions on structuring your datasets.
```
Comment (manonthegithub) [inline README.md:134]: Doesn't removing bullet points make it less emphasised? I would keep them, I think...

**Comment by manonthegithub** [comment]
@Debaditya-Som hey! I do not understand how do your changes address #92 What I see is that you just did some minor changes in the docs more like beutify... In general I don't see why would we use them...
#92 is about make a guide like https://dbpedia.gitbook.io/databus/guides/publish-guide for different storage options for files.

**Comment by Debaditya-Som** [comment]
> @Debaditya-Som hey! I do not understand how do your changes address #92 What I see is that you just did some minor changes in the docs more like beautify... In general I don't see why would we use them...
> #92 is about make a guide like https://dbpedia.gitbook.io/databus/guides/publish-guide for different storage options for files.

**Comment by Debaditya-Som** [comment]
> @Debaditya-Som hey! I do not understand how do your changes address #92 What I see is that you just did some minor changes in the docs more like beautify... In general I don't see why would we use them...
> #92 is about make a guide like https://dbpedia.gitbook.io/databus/guides/publish-guide for different storage options for files.

Okay so I need to create a guide like the one mentioned.
Yes actually this PR modifies the README.md file and enhances it to make it more appealing to the users and contributors. I'll surely work on making a guide like the one you have mentioned.
Thank you.

**Comment by manonthegithub** [comment]
I will close this pull request, feel free to submit a new one, but related to an open issue.

**Comment by Debaditya-Som** [comment]
@manonthegithub 
Okay I'll work on that issue accordingly
Can I open an issue regarding enchantment of the README.md files and work on that too as in my opinion the README.md could have been more appealing and user friendly that it is currently?

**Comment by manonthegithub** [comment]
@Debaditya-Som sure you are free to create new issues. It is just not our priority to work on prettifying readme atm (the changes you propose are quite arguable in my opinion and do not bring much value), so it may take some time (weeks). We were also planning to change the readme in the future as we already had some arguing on what should be left there and what moved to other parts of the documentation...

**Comment by manonthegithub** [comment]
@Debaditya-Som what is important also, so when you create pull requests or issues, not just to propose changes, but motivate what is wrong and how your changes improve current situation, and the motivation should be compelling/obvious to the others. So far, you did not motivate your changes except saying that in your opinion it is better (not really compelling). to motivate better first decribe the problem (like it is hard to navigate or see information or whatever) and then explain how your solution (pull request) solves it. Then it could be discussed.

File: README.md (hunk 2 above)
Comment (Debaditya-Som) [reply README.md:128]: Okayy got it

File: README.md (hunk 3 above)
Comment (Debaditya-Som) [reply README.md:134]: Actually it was like a "*" I have modified it as a bullet by giving : "-"

**Comment by Debaditya-Som** [comment]
> @Debaditya-Som what is important also, so when you create pull requests or issues, not just to propose changes, but motivate what is wrong and how your changes improve current situation, and the motivation should be compelling/obvious to the others. So far, you did not motivate your changes except saying that in your opinion it is better (not really compelling). to motivate better first decribe the problem (like it is hard to navigate or see information or whatever) and then explain how your solution (pull request) solves it. Then it could be discussed.

Surely, I will look forward to contribute in an issue that is more motivating. But in my opinion having a user friendly and enhanced README.md would attract more users to contribute in this particular project and make it more convenient for the and contributors to understand the motive and workflow of the particular project. So, that's the main reason behind this PR

**Comment by Debaditya-Som** [comment]
> @Debaditya-Som sure you are free to create new issues. It is just not our priority to work on prettifying readme atm (the changes you propose are quite arguable in my opinion and do not bring much value), so it may take some time (weeks). We were also planning to change the readme in the future as we already had some arguing on what should be left there and what moved to other parts of the documentation...

Alright then for the time being shall I create an issue regarding this and work on this minor issue? I know that its not of much importance but it will certainly have a positive impact on the whole project and repository

**Pull request closed without merging**